ws-nextjs-app/pages/api/local/[service]/[pageType]/[id]/local.api.test.ts

```diff
@@ -63,36 +65,52 @@ describe('API Routes', () => {
     });
   });
 
-  it('should return a 500 when the requested fixture does not exist', async () => {
+  it('should return AV Embeds data', async () => {
     await testApiHandler({
       pagesHandler,
       params: {
-        service: 'swahili',
-        pageType: 'invalidType',
-        id: 'u50853489',
+        service: 'russian',
+        pageType: 'av-embeds',
```

Comment:
```suggestion
        pageType: AV_EMBEDS,
```

Reply:
Slight consistency issue here as the `pageType` value is `avEmbeds`, whereas the folder is called `av-embeds` which is what we need to use here.

The reason pageType is `avEmbeds` is because we use it as query param for the BFF. I don't particularly like using hyphens in query params, even though it is valid, so maybe its ok?

Reply:
Oops I didn't spot that, let's just leave it. Hopefully won't be too confusing to maintain going forward 🤞